### PR TITLE
Update CHANGELOG for upcoming release

### PR DESCRIPTION
**Pull request by koeppea**
This PR is one of the last PRs to be merged before releasing.
It captures the highlights and possible caveats of the new release.

### CODE REVIEW

**Comment by koeppea** [comment]
Also copyright need to be updated

**Comment by sgeto** [comment]
Do you mean the GPL?

**Comment by koeppea** [comment]
No the copyright note contains the year.

**Comment by sgeto** [comment]
Added to checklist

**Comment by koeppea** [comment]
fa070d1: This what I found. Have I missed something else that is worth mentioning in the **CHANGELOG**?

**Comment by sgeto** [comment]
Now we only need the name.

**Comment by sgeto** [comment]
How do you intend to package the release? And under what compression/format?
If we use git, then we should make sure not to include such files as .travis.yml.

Another thing is that we can remove the need for bison and flex for people building from a release by adding the pre-processed files inside the tarball. I think libpcap does the same.
Thoughts?

**Comment by koeppea** [comment]
> How do you intend to package the release? And under what compression/format?

The same way the past release packages.

> If we use git, then we should make sure not to include such files as .travis.yml.

True.


> Another thing is that we can remove the need for bison and flex for people building from a release by adding the pre-processed files inside the tarball.

AFAIK bison and flex are required for compiling filters. So I don't see that we don't need these as being dependencies even for "runtime".


> I think libpcap does the same.

I think I don't get it. We provide the source code and contributing files to build and install ettercap regardless of which packaging system is being used. The downstream maintainer takes care of that, depending on the Distribution. To build and Ettercap, the libpcap is necessary.

@sgeto Any idea when we could finish the Fix CMake pull request, so to complete the "code maintenance" release? I've already quite some new features in the pipe but don't want to mess with the  upcoming release.

**Comment by koeppea** [comment]
Should we put #ettercap or #ettercap-project as IRC channel.
Myself is more hanging around in #ettercap-project.
What's your opinion?

**Comment by eaescob** [comment]
we need to add ssl redirect rules now :)

File: CHANGELOG [hunk 1]
```diff
@@ -5,6 +5,32 @@ Legend:
          !! bug fixed
 
 =========================================
+0.8.3-Belitto       2018mmdd
```
Comment (koeppea) [inline CHANGELOG]: Now when we know when to relase we can update the date which is then the commit to add the tag for 0.8.3.
I think the only thing left is to update the website.

**Comment by koeppea** [comment]
I think everything is ready for releasing Monday.
 1. New release is drafted.
 2. Source-only tarball attached.
 3. Website is prepared.
   a) website distinguish between 3 different display sizes and optimized for mobile devices
   b) content has been updated
   c) just the new digest hashes are outstanding
 4. Screencast to present the new version is ready for upload to our YouTube channel.
 5. E-Mail to maintainers drafted.

Any objections / comments?

**Comment by koeppea** [comment]
I'm still a bit unclear how ettercap.github.io and www.ettercap-project.org relate to each other.
@eaescob: Could you please explain? 
Thanks.

**Pull request merged**